tests/plugins/pastefromword/generated/_fixtures/Meta_generator/Excel/expected.html

```diff
@@ -52,9 +52,9 @@
 						</span>
 					</span>
 				</span>
-		</td>
-		<td style="background-color:#ffc000; height:26.25pt; vertical-align:bottom; white-space:nowrap; width:48pt">
-				<span style="font-size:11pt">
+			</td>
+			<td style="background-color:#ffc000; border-bottom:1px solid black; border-left:1px solid black; border-right:1px solid black; border-top:1px solid black; vertical-align:bottom; white-space:nowrap; width:64px">
```

Comment:
It's not lost. It rather never been there:
https://github.com/ckeditor/ckeditor-dev/blob/f0e034d40432cdb0f8e14c20538be9fac1a78b36/tests/plugins/pastefromword/generated/_fixtures/Meta_generator/Excel/excel2016/datatransfer.html#L207-L218
But I see that in the test case `tr` ascendant element have duplicated `height`. I'm wondering if it's worth to copy this property into a all child cells? The single first cell should be enough to keep row heigt. WDYT?

Reply:
I'm wondering if it can be broken by some styling involving e.g. `line-height`. In such case, if any other cell has height bigger than 35px, the whole row will be taller than intended. But it's rather an edge case.

I was just wondering, why the height disappeared.